docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -718,7 +714,7 @@ WITH (format='CSV',
       Defaults to ``0.05``.
 ```
 
-The table definition below specifies format Parquet, partitioning by columns
+The table definition below specifies to use Parquet files, partitioning by columns
```

Comment:
```suggestion
The table definition below specifies to use Parquet data files, partitioning by columns
```

Reply:
We just say Parquet files and ORC files everywhere else .. should I update across the Iceberg connector docs?

Reply:
> should I update across the Iceberg connector docs?

Probably not.
In the context above it should be obvious that we're speaking about data files.
Actually the end user shouldn't be concerned about the fact that there are as well metadata files.